dscript/pretrained.py

```diff
@@ -77,6 +85,35 @@ def get_state_dict(version="human_v1", verbose=True):
     return state_dict_fullname
 
 
+def retry(retry_count: int):
+    def decorate(func):
+        @wraps(func)
+        def retry_wrapper(*args, **kwargs):
+            attempt = 0
+            version = args[0]
+            while attempt < retry_count:
+                try:
+                    result = func(*args, **kwargs)
+                    return result
+                except RuntimeError as e:
+                    print(
```

Comment:
Please update this (and any other print statements) to use `utils.log`

Reply:
Sounds good!